services/121-service/src/events/events.service.ts

```diff
@@ -343,9 +378,11 @@ export class EventsService {
       'registrationProgramId',
       'personAffectedSequence',
       'lastMessageStatus',
-      'name',
       'inclusionScore',
+      // Add 'name' only if it's part of LogEntity; otherwise, remove it
```

Comment:
I think this comment in-code is actually a PR-comment/question...

So _is_ the `name`-property part of `LogEntity`? Then it should be added in the array above.

Is it _not_ part of it? Then this comment can also be removed, I'd think.

Reply:
Done